concordia/static/js/action-app/index.js

```diff
@@ -620,7 +620,8 @@ export class ActionApp {
         if (canEdit) {
             this.assetReservationURL = this.urlTemplates.assetReservation.expand(
                 {
-                    assetId: encodeURIComponent(asset.id)
+                    assetId: encodeURIComponent(asset.id),
+                    reserveId: encodeURIComponent(this.config.reserveId)
```

Comment:
Should we refactor this to be `reservationToken` to match the backend name?

Reply:
Oh this needs to be taken out. On it.